bloom/infra/spire_service.py

```diff
@@ -0,0 +1,155 @@
+from logging import getLogger
+from gql import Client
+from gql.transport.requests import RequestsHTTPTransport
+from requests import exceptions
+from collections.abc import MutableMapping
+
+from bloom.domain.vessel import Vessel
+from bloom.infra.spire_api_paging import Paging
+
+logger = getLogger()
+
+class SpireService:
+    def __init__(self) -> None:
+        self.transport = RequestsHTTPTransport(
+            url='https://api.spire.com/graphql',
+            # headers={'Authorization': f"Bearer {os.getenv('SPIRE_TOKEN')}"},
+            headers={'Authorization': 'Bearer U2hWhOiuDfAiVo8g07IIkMdzrDZtPGps'},
```

Comment:
Also this token was exposed in another branch so I also exposed it here to allow ppl to contribute. In any case it only allows us to query data on a single ship, so I'm assuming its temporary and for test purposes.